pulsar-edit/pulsar
# cirrus: Update Rolling upload token again

I must have accidentally regenerated the token an additional time after encrypting it last time, such that the encrypted version was already inactive (regenerated over) by the time the PR went up to switch to it.

Fingers crossed I've done it correctly this time.

Follow-up to https://github.com/pulsar-edit/pulsar/pull/936.


### Validation Steps

~~I can do a test cron build of Cirrus with this token and see if it uploads?~~

Update: It worked! ✅ 

See:
- https://github.com/pulsar-edit/pulsar-rolling-releases/releases/tag/v1.114.2024031823
- https://cirrus-ci.com/task/5900157967925248?logs=rolling_upload#L22

```
Uploading local binaries to rolling release repo: /tmp/cirrus-ci-build/binaries/Pulsar-1.114.2024031823-arm64.AppImage,/tmp/cirrus-ci-build/binaries/pulsar-1.114.2024031823-arm64.tar.gz,/tmp/cirrus-ci-build/binaries/pulsar-1.114.2024031823.aarch64.rpm,/tmp/cirrus-ci-build/binaries/pulsar_1.114.2024031823_arm64.deb
Releases published successfully: https://github.com/pulsar-edit/pulsar-rolling-releases/releases/tag/v1.114.2024031823
```

Unrelated CI failure on Cirrus macOS, something about not being able to symlink Homebrew's coopy of yarn... But that's out of scoep for this PR, the token update worked this time 🎉
